README.md

```diff
@@ -75,7 +75,19 @@ Note: Neovide requires neovim version 0.4 or greater.
 
 ### Linux
 
-Note: Neovide has been successfully built on other destros but this reportedly works on ubuntu.
+#### Arch Linux
+
+There is an [AUR package for neovide](https://aur.archlinux.org/packages/neovide).
```

Comment:
This is awesome, but I don't want to add it to the readme until I know who is uploading them and backing them. Is it you?

Reply:
No I'm not the maintainer. [The PKGBUILD](https://aur.archlinux.org/cgit/aur.git/tree/PKGBUILD?h=neovide) has the maintainer as, `peeweep <peeweep at 0x0 dot ee>` ([peeweep](https://aur.archlinux.org/account/peeweep)).

Reply:
Oh one other thing to note just in case this influences your position. The AUR doesn't distribute binary packages, only the `PKGBUILD` file describing how to build the package. Folks running Arch will each need to build neovide according to the `PKGBUILD`.